ql/src/java/org/apache/hadoop/hive/ql/ddl/table/partition/add/AlterTableAddPartitionDesc.java

```diff
@@ -234,7 +233,7 @@ public void setWriteId(long writeId) {
 
   @Override
   public String getFullTableName() {
-    return AcidUtils.getFullTableName(dbName,tableName);
+    return AcidUtils.getFullTableName(dbName, tableName);
```

Comment:
All changes in this file are out of scope of the PR.  Please revert.

Reply:
Removed.